graphql/task_resolver.go

```diff
@@ -269,7 +269,7 @@ func (r *taskResolver) DependsOn(ctx context.Context, obj *restModel.APITask) ([
 
 	t, err := obj.ToService()
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting service model for APITask %s: %s", *obj.Id, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting service model for APITask %s: %s", *obj.Id, err.Error()))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting service model for APITask '%s': %s", *obj.Id, err.Error()))
```